test-fixture/src/sim/taildrop.rs

```diff
@@ -31,65 +118,67 @@ pub struct TailDrop {
     rate: usize,
     /// The depth of the queue, in bytes.
     capacity: usize,
+    /// Whether to apply ECN markings.
+    ecn: bool,
 
     /// A counter for how many bytes are enqueued.
     used: usize,
     /// A queue of unsent bytes.
     queue: VecDeque<Datagram>,
     /// The time that the next datagram can enter the link.
-    next_deque: Option<Instant>,
+    next_deque: Option<(Instant, u32)>,
```

Comment:
Comment doesn't say what the new `u32` is for. Is it the removed `sub_ns_delay`?

Reply:
Yeah, it was cleaner that way.